src/test/ui/underscore-lifetime/in-fn-return-illegal.stderr

```diff
@@ -2,13 +2,13 @@ error[E0106]: missing lifetime specifier
   --> $DIR/in-fn-return-illegal.rs:5:30
    |
 LL | fn foo(x: &u32, y: &u32) -> &'_ u32 { loop { } }
-   |                              ^^ expected named lifetime parameter
+   |           ----     ----      ^^ expected named lifetime parameter
    |
    = help: this function's return type contains a borrowed value, but the signature does not say whether it is borrowed from `x` or `y`
 help: consider introducing a named lifetime parameter
    |
-LL | fn foo<'lifetime>(x: &u32, y: &u32) -> &'lifetime u32 { loop { } }
-   |       ^^^^^^^^^^^                       ^^^^^^^^^
+LL | fn foo<'a>(x: &'a u32, y: &'a u32) -> &'a u32 { loop { } }
+   |       ^^^^    ^^^^^^^     ^^^^^^^      ^^
```

Comment:
I'm a bit confused, why are the `^`s for the `'a` additions in the reference types covering the entire type?

Either way, this is looking great!

Reply:
Simply because of the `Span` I'm using comes from the `hir::Ty` directly. We do not have a span pointing only at the `&`s, but I can synthesize one, I just find doing so to be problematic way too often to be worth it.

> Either way, this is looking great!

You should see it in full 16 colors! (Well, 6 really.)

<img width="1051" alt="Screen Shot 2020-01-28 at 2 14 12 PM" src="https://user-images.githubusercontent.com/1606434/73310564-5fd93300-41d9-11ea-8661-cbeba0e5446b.png">